fjord/feedback/static/css/generic_feedback.less

```diff
@@ -135,19 +135,19 @@ body.html-rtl header .header-content .left-side {
 
 #back-button-container {
   display: none;
-  height: 30px;
-  margin: 0;
+  height: 38px;
+  margin: 0px 0px 0px 0px;
```

Comment:
In CSS, `margin: 0;` is shorthand for `margin: 0 0 0 0;`. Also, with `0` you don't have to specify the unit.